src/components/tab/tab-styles.jsx

```diff
@@ -0,0 +1,106 @@
+const borderHeight = 2;
+
+export default theme => {
+  const { palette, typography, mixins } = theme;
+
+  return {
+    root: {
+      padding: [0, 10],
+
+      '&:first-child': {
+        paddingLeft: 0,
+      },
+
+      '&:last-child': {
+        paddingRight: 0,
+      },
+    },
+
+    tab: {
+      position: 'relative',
+      display: 'block',
+      fontSize: '12px',
+      color: palette.text.muted,
+      fontFamily: typography.primary.fontFamily,
+      fontWeight: typography.fontWeightSemiBold,
+      paddingTop: 5,
+      paddingBottom: borderHeight + 2,
+      textTransform: 'uppercase',
+      letterSpacing: 0.5,
+      whiteSpace: 'nowrap',
+      cursor: 'pointer',
+
+      'button&': {
+        background: 'none',
+        border: 0,
+      },
+
+      'a&': {
```

Comment:
same here

Reply:
& is the parent selector, in this case we are selecting all a tags that also has the parent class on it. Same for your button comment :)